### PR TITLE
Bugfix to move 'GOOGLE_SPEECH_URI' definition

### PR DESCRIPTION
**Pull request by slay2k**
I debugged the issue of why the extension was hanging on "Searching...", especially in other languages like Spanish. This seems to have been the problem (variable undefined in scope of use), so I'm attaching a quick fix.

Tested and it works for me.

### CODE REVIEW

**Comment by meetDeveloper** [comment]
Is the meaning coming now?

**Comment by meetDeveloper** [comment]
Oh, noticed the mistake now, it slipped somehow.

File: background/background.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+var GOOGLE_SPEECH_URI = 'https://www.google.com/speech-api/v1/synthesize';
+
+
```
Comment (meetDeveloper) [inline background/background.js:3]: Can you use `const` and also just have one new line instead of two.

**Comment by meetDeveloper** [comment]
Added a comment, little change is required and will merge it.

File: background/background.js (hunk 1 above)
Comment (slay2k) [reply background/background.js:2]: Done.

**Comment by meetDeveloper** [comment]
Screenshot 2020-07-18 at 1 10 38 AM

I have updated the extension, it will be available shortly. Thank you for your contribution. I appreciate it a lot. @slay2k

**Comment by meetDeveloper** [comment]
@slay2k Updated extension v1.0.6 is now available on website, can you check it once? Sorry for the inconvenience.

**Comment by slay2k** [comment]
Yup, works now!

**Pull request merged**